.gitignore

```diff
@@ -6,6 +6,7 @@ coverage
 
 
 backend/static
+backend/uploads
```

Comment:
Should be fine for now but there may come a time and place where we may need to consider using a service like Minio or equivalent for temporary file storage and management instead of doing it directly on the pod.

Reply:
yes, I'll make sure next update that I add all the env vars and update the readme to reflect.